src/openforms/authentication/contrib/digid/tests/test_signicat_integration.py

```diff
@@ -0,0 +1,608 @@
+        # select substantial LOA
+        sim_form_2["loa"] = "loa4"
+        auth_response_2 = session.request(
+            sim_method_2, sim_action_url_2, data=sim_form_2, allow_redirects=False
+        )
+
+        # redirect back to our acs
+        self.assertEqual(auth_response_2.status_code, 302)
+        acs_url_2 = furl(auth_response_2.headers["location"])
+        # prep the url for django testclient consumption
+        acs_url_2.remove(netloc=True, scheme=True)
+        acs_response_2 = self.client.get(acs_url_2.url, follow=True)
+
+        # we are logged in
+        self.assertIn(FORM_AUTH_SESSION_KEY, self.client.session)
+        # and we end up starting the form
+        self.assertEqual(acs_response_2.resolver_match.view_name, "core:form-detail")
+        self.assertEqual(acs_response_2.resolver_match.kwargs["slug"], "slurm")
+
+    def test_test_key_material_is_not_fishing_bait(self):
+        config = DigidConfiguration.get_solo()
+        config.base_url = "https://sharkbait.example.com"
+        config.save()
+
+        def reset_config():
+            config.base_url = config.entity_id = "https://localhost:8000"
+            config.save()
+
+        self.addCleanup(reset_config)
+
+        session: requests.Session = requests.session()
+        form = FormStepFactory.create(
+            form__slug="slurm",
+            form__authentication_backends=[PLUGIN_ID],
+            form__authentication_backend_options={
+                PLUGIN_ID: {"loa": DigiDAssuranceLevels.base}
+            },
+            form_definition__login_required=True,
+        ).form
+
+        login_url = reverse(
+            "authentication:start", kwargs={"slug": form.slug, "plugin_id": PLUGIN_ID}
+        )
+        form_path = reverse("core:form-detail", kwargs={"slug": form.slug})
+        form_url = furl("http://testserver/") / form_path
+
+        our_faux_redirect = self.client.get(f"{login_url}?next={form_url}", follow=True)
+        # do the JS submit to get redirected to signicat broker
+        method, redirect_url, form_values = _parse_form(our_faux_redirect)
+        session.request(method, redirect_url, data=form_values)
+        select_digid_sim = SIGNICAT_BROKER_BASE / "authn/simulator/selection/digid"
+        sim_response = session.get(select_digid_sim)
+        sim_method, sim_action_url, sim_form = _parse_form(sim_response)
+        auth_response = session.request(
+            sim_method, sim_action_url, data=sim_form, allow_redirects=False
+        )
+        # not redirected
+        self.assertFalse(auth_response.ok)
+
+
+# class SignicatWithEncryptedAssertionsTests(SignicatDigiDIntegrationTests):
+#     """This doens't work; after changing our DigidConfiguration, we need to send our updated
+#     SAML metadata to the broker (Signicat)
+#
+#     To test this we'd need to configure a separate entityID
+#     (perhaps with a different certificate/key pair) and set it up with Signicat
+#     """
```

Comment:
afaik these model fields only exist to be able to generate the metadata and exchange it out of band